tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -0,0 +1,179 @@
+""" This is a test of the ProxyDB
+
+    It supposes that the DB is present and installed in DIRAC
+"""
+
+# pylint: disable=invalid-name,wrong-import-position,protected-access
+import os
+import sys
+import unittest
+
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
+
+from DIRAC import gLogger, gConfig
+from DIRAC.Core.Utilities.CFG import CFG
+from DIRAC.FrameworkSystem.DB.ProxyDB import ProxyDB
+
+
+diracTestCACFG = """
```

Comment:
Could you reuse the same test proxies and everything that is already in integration branch ? It would be much easier. Once ported to integration, we can just point to the existing location

Reply:
The v7 integration test/certs has been added to use it in an existing test, I have not found any ProxyDB tests or that can be use for ProxyDB testing. Could you give me test files links from v7 that needed to integrate to this PR or what thing additionally need to test

Reply:
They are not tests for the proxyDB but for the X509 objects that carries your PEM identity. It is all the `Test_X509*` files in https://github.com/DIRACGrid/DIRAC/tree/integration/Core/Security/test 
It would be interesting that you run these tests against a *clean proxy* that you get from a provider. 
Of course, these tests were designed to compare and run against PyGSI and M2Crypto for the existing cases, so probably you need to extend them

Reply:
Tests for the DIRAC CA are included into #4070. This is the main use case. Testing RCAuth proxies will anyway require mocking-up the RCAuth proxy and will be reduced to the similar use case as DIRAC CA proxies